.github/workflows/deploy.yml

```
@@ -0,0 +1,43 @@
+name: Сборка VEGA-2
+on:
+  push:
+    branches: master
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  build:
+    name: Сборка
```

в интерфейсе гитхаба будет понятнее так
```suggestion
    name: Сборка и деплой VEGA-2
```